test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/IntegratedTriggersAndBindingsTests.cs

```diff
@@ -177,6 +189,147 @@ public static IHostBuilder ConfigureGeneratedFunctionMetadataProvider(this IHost
                     expectedOutput);
             }
 
+            [Theory]
+            [InlineData(LanguageVersion.Latest)]
+            public async void FunctionsMultipleOutputBindingWithActionResult(LanguageVersion languageVersion)
+            {
+                string inputCode = """
+                using System;
+                using System.Diagnostics.CodeAnalysis;
+                using System.Net;
+                using Microsoft.Azure.Functions.Worker;
+                using Microsoft.Azure.Functions.Worker.Http;
+                using Microsoft.AspNetCore.Http;
+                using Microsoft.AspNetCore.Mvc;
+
+                namespace FunctionApp
+                {
+                    public static class FunctionsMultipleOutputBindingWithActionResult
+                    {
+                        [Function(nameof(FunctionsMultipleOutputBindingWithActionResult))]
+                        public static MyOutputType Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+                            FunctionContext context)
+                        {
+                            throw new NotImplementedException();
+                        }
+
+                        [Function("OutputTypeHttpHasTwoAttributes")]
+                        public static MyOutputType2 Test([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+                            FunctionContext context)
+                        {
+                            throw new NotImplementedException();
+                        }
+                    }
+
+                    public class MyOutputType
+                    {
+                        [QueueOutput("functionstesting2", Connection = "AzureWebJobsStorage")]
+                        public string Name { get; set; }
+
+                        [HttpResult]
+                        public IActionResult HttpResponse { get; set; }
+                    }
+
+                    public class MyOutputType2
```

Comment:
Is it valid to write something like below? (even with legacy generator approach / in-proc model)

```csharp
public class MyOutputTypeToMultipleDestinations
{
    [QueueOutput("functionstesting2", Connection = "AzureWebJobsStorage")]
    [ServiceBusOutput("my-queue", Connection = "ServiceBusConnection")]
    public string Name { get; set; }


    [HttpResult]
    public IActionResult HttpResponse { get; set; }
}
```
Does this cause a diagnostic error today? Do we have a test for that?

Reply:
That is not valid and results in a diagnostic error - [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/docs/analyzer-rules/AZFW0005.md) is the doc for it, [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/add69528ff7127b1db4cb8df89a2a3fff9bb4788/test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/DiagnosticResultTests.cs#L54) is a test for it, and [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/add69528ff7127b1db4cb8df89a2a3fff9bb4788/sdk/Sdk/FunctionMetadataGenerator.cs#L332) is where this same check/error is thrown in the legacy generator.